### ISSUE TITLE
Example App vs Create-dogstack-app versioning?

### ISSUE THREAD

**Issue by danalexilewis** [root-systems/create-dogstack-app#1]
I was going to suggest a change to the html cdns - and than wasnt sure whether it belonged here or on the example app. 

It made me wonder if this repo should have versioning for template files and the example app should use these?

Thoughts?

**Comment by iainkirkpatrick** [comment]
yeah there's def something to consider with what the purpose of this repo is vs the purpose of `dogstack-example`. I guess my initial thought on it was that this app would become the go-to for starting a new dogstack app, probably with some easy-deployment stuff included, and that the example repo might be retired... 

but haven't thought about it that much, wonder what value i'm missing in the example repo once this one is up to scratch.

**Comment by ahdinosaur** [comment]
> but haven't thought about it that much, wonder what value i'm missing in the example repo once this one is up to scratch.

i'm not sure how a generator replaces a demo app. it's true that the current example isn't much more than a template, but i was hoping we could do something fun with it. or maybe the demo app is something much more complete, like a Tarot or Astrology app for giggles.
